pkg/storaged/storage.scss

```diff
@@ -400,14 +400,22 @@ td.storage-action {
     margin-right: 1em;
 }
 
+.sigkey-heading {
+    color: var(--pf-global--Color--dark-100);
+    font-size: var(--pf-global--FontSize--sm);
+    margin: var(--pf-global--spacer--md) 0 0;
+}
+
 .sigkey-hash {
     font-family: monospace;
     font-size: 140%;
 }
 
 .inline-pre {
+    background: var(--pf-global--BorderColor--300);
```

Comment:
OK Patternfly does not style ` <pre> ` with grey background etc.
Instead for code blocks or text areas they provide their own components - see https://www.patternfly.org/v4/components/code-block 

As this is overriding default PF styles I think this is not correct do here - as we 're trying to move away from custom stylesheets.